services/src/main/java/org/fao/geonet/services/schema/Info.java

```diff
@@ -182,10 +182,20 @@ public static Element getHelp(SchemaManager scm, Element elem, String fileName,
         if (result == null) {
         	result = checkEntries(scm, schema, entries, parent, name, isoType, false);
         }
+
+        // if we're in a schema that supports profiles (eg. iso19139, iso19115-2 or iso19115-3), then check the base schema
```

Comment:
Maybe better to follow the dependencies between schemas - ie. for example, a schema that depends on iso19139 is iso19139.anzlic - also helps for iso19115-2 profiles that want to check iso19115-2 and then as a last resort iso19139